### DISCUSSION TITLE
Automatic Cluster Labeling

### DISCUSSION THREAD

**Discussion by rizkiamandaputri** [MaartenGr/BERTopic#1915] [Q&A]
Can BERTopic create automatic cluster labeling?
How can do it?
Thank you.

**Comment by MaartenGr** [comment]
Have you checked out the [documentation](https://maartengr.github.io/BERTopic/)? You can find many guides there with respect to automatic labeling! One good source is either one of the Google Colab tutorials or the guides [here](https://maartengr.github.io/BERTopic/getting_started/representation/llm).
